doc/source/configuration.rst

```diff
@@ -170,9 +169,12 @@ The Nextflow configuration file is used to configure various settings for the *k
     
       There are 11 processes in the pipeline, 6 of which (``checkConfig``, ``setupCorrection``, ``setParalogAnalysis``, ``setOrthologAnalysis``,  ``doRateCorrection`` and ``drawTree``) are by default run locally because they execute minimal calculations. The remaining 5 processes (``estimatePeak``, ``plotOrthologDistrib``, ``paralogsAnalyses``, ``wgdParalogs`` and ``wgdOrthologs``) are instead run by default on a cluster, if available, and can be configured under this section of the Nextflow configuration file. ``wgdParalogs`` and ``wgdOrthologs`` are the most computationally demanding and it is advised to assign them a higher computational power than the other processes. If available, we suggest to configure about 10 CPUs/cores/slots/threads and about 20GB memory (or, on average, about 2GB per configured CPU) for each of these two processes.
     
-      Examples of available settings are (for a complete list see the `Nextflow documentation <https://www.nextflow.io/docs/latest/process.html#process-directives>`__):
+      Cluster settings can be tailored on your own machine type through the use of Nextflow directives (for a complete list see the `Nextflow documentation <https://www.nextflow.io/docs/latest/process.html#process-directives>`__), such as:
     
-    	* **clusterOption** any native configuration option accepted by your cluster submit command. You can use it to request non-standard resources or use settings that are specific to your cluster and not supported out of the box by Nextflow.
+        * **cpus** sets the number of CPUs/cores/slots/threads, e.g. ``8`` [Default if not set: 1]
```

Comment:
Maybe we should stress more that this directive is mandatory or needs to be set if one wants more than 1 CPU?

Reply:
I can add a short sentence to suggest to use multiple threads for the two wgd processes.

Reply:
The comment was more about that it's required to use this directive to set more than 1 CPU (as opposed to e.g. `-pe serial 8`).

Reply:
I tried it out again: cpus can be without quote, but memory and time need quotes (perhaps because they're a string such as 3GB or 1h, while cpus is a number). I'll remove the quotes from cpus!